nntrainer/layers/layer_node.cpp

```diff
@@ -186,6 +195,16 @@ bool LayerNode::setProperty(const std::string &key, const std::string &value) {
 
   PropertyType type = static_cast<PropertyType>(parseLayerProperty(key));
   switch (type) {
+  case PropertyType::output_shape: {
+    std::vector<TensorDim> output_dim = init_context.getOutputDimensions();
+    if (getNumOutputs() == 0)
+      output_dim.resize(1);
+
+    // for (auto iter = output_dim.cbegin();iter!= output_dim.cend();iter++){
+    // NYI
+    // }
+
+  } break;
```

Comment:
FYI) This line won't reach as line 141 will already parse output_shape. But looks like this logic perhaps can be moved to `finalize()`